cfn-resources/cloud-backup-snapshot-export-job/test/cfn-test-create-inputs.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+# cfn-test-create-inputs.sh
+#
+# This tool generates json files in the inputs/ for `cfn test`.
+#
+
+set -o nounset
+set -o pipefail
+set -x
+
+function usage {
+    echo "usage:$0 <project_name>"
+}
+
+if [ "$#" -ne 1 ]; then usage; fi
+if [[ "$*" == help ]]; then usage; fi
+
+rm -rf inputs
+mkdir inputs
+region="us-east-1"
+
+#project_id
+projectName="${1}"
+
+
+projectId=$(atlas projects list --output json | jq --arg NAME "${projectName}" -r '.results[] | select(.name==$NAME) | .id')
+if [ -z "$projectId" ]; then
+    projectId=$(atlas projects create "${projectName}" --output=json | jq -r '.id')
+    echo -e "Created project \"${projectName}\" with id: ${projectId}\n"
+else
+    echo -e "FOUND project \"${projectName}\" with id: ${projectId}\n"
+fi
+
```

Comment:
Can you use this code to create cluster which checks status and wait instead of waiting for 1200 seconds?

https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/77799a4368e377246ce0f3b52c2ef4e9f99f5035/cfn-resources/cloud-backup-schedule/test/cfn-test-create-inputs.sh#L35